app.py

```diff
@@ -9,7 +13,9 @@
 @app.route("/", methods = ["GET", "POST"])
 def homepage():
     if request.method == "GET":
-        return render_template("index.html")
+    	url = "https://www.googleapis.com/civicinfo/v2/representatives?address=10027&key=AIzaSyAalDN2dXO26te2Soy9gAsOU_wvSlYghVg"
```

Comment:
Could we have a comment about what this URL is?

Also, plain-text keys are generally :no_good: from a security perspective, although we can probably get away with it for now. The usual solution is to store them in a configuration file that's not committed to git and manually distribute it.

If you do choose to ignore the security concerns, at least add a `# TODO: Plaintext API Key` as a reminder so we can remember it when we move towards deployment.